Added some Energy generators (#84)
Slimefun/Wiki: pages/Bio-Reactor.md

The Bio Reactor is a type of [Energy Generator](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines#energy-generation). 

## Obtaining
A Bio Reactor can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).

## Usage

> A Bio Reactor must be placed within 7 blocks of an [Energy Regulator](https://github.com/Slimefun/Slimefun4/wiki/Energy-Regulator) or a [Capacitor](https://github.com/Slimefun/Slimefun4/wiki/Energy-Capacitors) in a cross format (not diagonally), and provided with the appropriate fuel.<br>

Reviewer: * or an Energy Connector

> The Bio Reactor is a type of [electricity](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) generator. 

Reviewer 2: ```suggestion
The Bio Reactor is a type of [Energy Generator](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines#energy-generation). 
```
Keep naming and update link to the subcategory there (consistency with other files)

## Obtaining
A Bio Reactor can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).

## Usage

> A Bio Reactor must be placed within 7 blocks of an [energy regulator](https://github.com/Slimefun/Slimefun4/wiki/Energy-Regulator) or a [capacitor](https://github.com/Slimefun/Slimefun4/wiki/Energy-Capacitors) in range of one (in a straight line; no diagonals), and provided with the appropriate fuel.<br>

Reviewer 2: ```suggestion
A Bio Reactor must be placed within 7 blocks of an [Energy Regulator](https://github.com/Slimefun/Slimefun4/wiki/Energy-Regulator) or a [Capacitor](https://github.com/Slimefun/Slimefun4/wiki/Energy-Capacitors) in a cross format (not diagonally), and provided with the appropriate fuel.<br>
```

Once it is placed, right-click it and place fuel in its inventory (two slots on the left).

## Tiers

> There is only one tier of Bio Reactor.

Reviewer 2: ```suggestion
The Bio Reactor only has one tier.
```

| Tier | Energy | Buffer |
| ---- | --- | ------ |
| I | 8 J/s | 128 J |

## Fuel

> This generator can be fueled with various vegetation.

Reviewer 2: ```suggestion
This generator can be fueled with various compostable items.
```
I feel like vegetation is wrong wording here, I thought flora was better but apparently theyre synonyms
(vegetation is the growth of plants over an area)
This is my suggestion towards it but if you have something

Author: To be fair, I had a bit of a problem with this - neither vegetation/flora works well (because Bio-Reactor uses bones, spider eyes, etc for fuel as well) nor compostable items (because again, some items won't work with either vanilla or slimefun composter).
I'm kind of inclined to just blank-card it with "various [organic?] items".

| Fuel | Burning time | Total |
| ---- | ------------ | --------------------- 
| Rotten Flesh | 2s |  16 J
| Spider Eye | 2s |  16 J
| Bone | 2s |  16 J
| String | 2s |  16 J
| Apple | 3s |  24 J
| Melon Slice | 3s |  24 J
| Melon | 27s |  216 J
| Pumpkin | 3s |  24 J
| Pumpkin Seeds | 3s |  24 J
| Melon Seeds | 3s |  24 J
| Wheat | 3s |  24 J
| Wheat Seeds | 3s |  24 J
| Carrot | 3s |  24 J
| Potato | 3s |  24 J
| Sugar Cane | 3s |  24 J
| Nether Wart | 3s |  24 J
| Any Mushrooms | 2s |  16 J
| Vines | 2s |  16 J
| Cactus | 2s |  16 J
| Lily Pad | 2s |  16 J
| Chorus Fruit | 8s |  64 J
| Kelp | 1s |  8 J
| Dried Kelp | 2s |  16 J
| Dried Kelp Block | 20s |  160 J
| Seagrass | 1s |  8 J
| Sea Pickle | 2s |  16 J
| Bamboo | 2s |  16 J
| Sweet Berries | 2s |  16 J
| Any Flowers | 1s |  8 J
| Honeycomb | 4s |  32 J
| Honeycomb Block | 40s |  320 J
| Shroomlight | 4s |  32 J
| Strange Nether Goo | 16s |  128 J
| Any Leaves | 1s |  8 J
| Any Saplings | 1s |  8 J
| Any Corals | 2s |  16 J
| Any Coral Blocks | 2s |  16 J

> 

Reviewer 2: Redundant extra lines (keep one)
